docs/DOCKER.md

```diff
@@ -112,17 +82,16 @@ docker run \
 --network "elastic" \
 --tty \
 --rm \
-docker.elastic.co/enterprise-search/elastic-connectors:8.7.0.0-SNAPSHOT \
+docker.elastic.co/enterprise-search/elastic-connectors:8.10.0.0-SNAPSHOT \
 /app/bin/elastic-ingest \
 -c /config/config.yml
 ```
 
 You might need to adjust some details here:
+
 - `-v ~/connectors-python-config:/config \` - replace `~/connectors-python-config` with the directory that you've created in step 2 if you've chosen a different name for it.
 - `--network "elastic"` - replace `elastic` with the network that you've created in step 1 if you've chosen a different name for it.
-- `docker.elastic.co/enterprise-search/elastic-connectors:8.7.0.0-SNAPSHOT` - adjust the version for the connectors to match your Elasticsearch deployment version. 
-  - For Elasticsearch of version 8.7 you can use `elastic-connectors:8.7.0.0`for a stable revision of the connectors, or `elastic-connectors:8.7.0.0-SNAPSHOT` if you want the latest nightly build of the connectors (not recommended).
-  - If you are using nightly builds, you will need to run `docker pull docker.elastic.co/enterprise-search/elastic-connectors:8.7.0.0-SNAPSHOT` before starting the service. This ensures you're using the latest version of the Docker image.
-- `-c /config/config.yml` - replace `config.yml` with the name of the config file you've put in the directory you've created in step 2. 
-  - Normally, if you run a single instance of the Connector Service you can use `config.yml`. 
-  - If you're planning to run multiple versions of the service, you can put configs for different service versions in a single directory and update this setting to point to different config files.
+- `docker.elastic.co/enterprise-search/elastic-connectors:8.10.0.0-SNAPSHOT` - adjust the version for the connectors to match your Elasticsearch deployment version.
+  - For Elasticsearch of version 8.7 you can use `elastic-connectors:8.10.0.0`for a stable revision of the connectors, or `elastic-connectors:8.10.0.0-SNAPSHOT` if you want the latest nightly build of the connectors (not recommended).
```

Comment:
Versions are mismatched (and possibly another argument for not including exact versions in the docs 😄)
```suggestion
  - For Elasticsearch of version 8.10 you can use `elastic-connectors:8.10.0.0`for a stable revision of the connectors, or `elastic-connectors:8.10.0.0-SNAPSHOT` if you want the latest nightly build of the connectors (not recommended).
```